examples/python-dataflow/dataflow_without_webcam.yml

```diff
@@ -5,7 +5,8 @@ communication:
 nodes:
   - id: no_webcam
     custom:
-      source: ./no_webcam.py
+      source: python
```

Comment:
Is there any specific reason for that change?

Reply:
it's just cosmetics. But I will remove it.